quickstarts/aws/amazon-timestream/config.yml

```
@@ -0,0 +1,31 @@
+slug: amazon-timestream
+description: |-
+  ## What is Amazon Timestream?
+  Amazon Timestream is a fast, scalable, and serverless time-series database service that makes it easier to store and analyze trillions of events per day up to 1,000 times faster. Amazon Timestream automatically scales up or down to adjust capacity and performance, so that you don’t have to manage the underlying infrastructure.
```

```suggestion
  Amazon Timestream is a serverless scalable time-series database service to store and analyze up to trillions of events per day. It automatically scales up or down to adjust capacity and performance.
```

The "1000 times faster" seems a bit marketingy for AWS (faster than what, for example?).  You might consider adjusting the copy to be more of a neutral description of Amz Timestream

---

@seemantk, addressed, I have updated the description.